Resources/Private/BackendModule/src/Views/ListView/DocumentNodeListViewItemProperty.tsx

```
@@ -68,9 +70,17 @@ export default class DocumentNodeListViewItemProperty extends PureComponent<Docu
                         updateItemProperty={(value: string, state: ListItemPropertyState) => this.props.updateItemProperty(value, state)}
                     />
                 )
+            case 'NEOSidekick.AiAssistant/Inspector/Editors/ImageAltTextEditor':
+            case 'NEOSidekick.AiAssistant/Inspector/Editors/ImageTitleEditor':
+                if (!propertySchema?.ui?.inspector?.editorOptions?.imagePropertyName) {
+                    return <div style={{background: '#ff460d', color: '#fff', padding: '8px'}}>Incorrect YAML Configuration: Image Text Editor requires an editorOption <i>imagePropertyName</i></div>;
+                }
+
+                let module = propertySchema.ui.inspector.editor === 'NEOSidekick.AiAssistant/Inspector/Editors/ImageAltTextEditor' ? 'image_alt_text' : 'image_title';
+                finalPropertySchema = createMagicTextAreaEditorPropsForImageTextEditor(propertySchema, module);
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)